pkgs/dart_mcp_server/CHANGELOG.md

```diff
@@ -1,66 +1,67 @@
 # 0.1.1 (Dart SDK 3.10.0) - WIP
 
-* Change tools that accept multiple roots to not return immediately on the first
```

Comment:
Changelog changes should be a separate PR, will this now be standard with other pkgs from us?  Do we have this documented somewhere?  With the IntelliJ plugins, we are standardizing to https://keepachangelog.com/en/1.1.0/ (shout out to @helin24)